util/proxy.go

```diff
@@ -0,0 +1,49 @@
+package util
+
+import (
+	"context"
+	"net"
+	"net/url"
+	"sync"
+
+	"github.com/go-sql-driver/mysql"
+	netproxy "golang.org/x/net/proxy"
+
+	"github.com/pingcap/tipocket/pkg/test-infra/fixture"
```

Comment:
I think pkg `util` should not rely on the pkg `pkg/test-infra/fixture`?

Reply:
resolved in 71fce7ef35dd6b48107126a6ebfd0b79a716b3f0.